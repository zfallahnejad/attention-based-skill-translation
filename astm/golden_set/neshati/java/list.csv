22656,list
571407,list
1679863,list
1221571,list
869736,list
367273,list
829571,list
992484,list
1103872,list
1065197,list
335858,list
203907,list
1357341,list
1707091,list
157882,list
