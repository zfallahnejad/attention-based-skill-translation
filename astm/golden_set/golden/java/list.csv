22656,52,44,0.5416666666666666
571407,31,31,0.5
1679863,30,14,0.6818181818181818
1221571,21,24,0.4666666666666667
869736,18,18,0.5
367273,16,15,0.5161290322580645
829571,15,21,0.4166666666666667
992484,14,17,0.45161290322580644
335858,13,19,0.40625
1065197,13,11,0.5416666666666666
203907,13,19,0.40625
1103872,13,17,0.43333333333333335
1357341,12,7,0.631578947368421
1707091,11,13,0.4583333333333333
157882,11,8,0.5789473684210527
522444,10,11,0.47619047619047616
